mergify_engine/config.py

```diff
@@ -259,6 +260,7 @@ def ApplicationAPIKeys(v: str) -> typing.Dict[str, ApplicationAPIKey]:
 BOT_USER_LOGIN: str
 STORAGE_URL: str
 STREAM_URL: str
+QUEUE_URL: str
```

Comment:
If we go with QUEUE_URL, we should use the same terminology everywhere eg: redis_queue, create_yaaredis_queue, ...

Reply:
Gonna refactor everything from "train" to "queue" seems indeed more appropriate